armi/physics/fuelCycle/fuelHandlers.py

```diff
@@ -727,22 +727,38 @@ def swapAssemblies(self, a1, a2):
             :id: I_ARMI_SHUFFLE_MOVE
             :implements: R_ARMI_SHUFFLE_MOVE
 
+            For the two assemblies that are passed in, call to their :py:meth:`~armi.reactor.assemblies.Assembly.moveTo`
+            methods to transfer their underlying ``spatialLocator`` attributes to
+            each other. This will also update the ``childrenByLocator`` list on the
+            core as well as the assembly parameters ``numMoves`` and ``daysSinceLastMove``.
+
         .. impl:: User-specified blocks can be left in place and not moved.
             :id: I_ARMI_SHUFFLE_STATIONARY0
             :implements: R_ARMI_SHUFFLE_STATIONARY
 
```

Comment:
```suggestion
            :implements: R_ARMI_SHUFFLE_STATIONARY

            This is the implementation of the stationary requirement for the 
            within-core swap.
```

Reply:
I would argue this text is so central, it should be covered by the "title" of the implementation (that first line with the `.. impl::` tag). So, perhaps this sentence is a hint that you want tweak the title?

Reply:
https://github.com/terrapower/armi/pull/1589/commits/39d55597b00ba43df7be17ad69db9aedcdc0889d